src/cascadia/TerminalApp/App.cpp

```diff
@@ -4,6 +4,7 @@
 #include "pch.h"
 #include "App.h"
 #include <winrt/Microsoft.UI.Xaml.XamlTypeInfo.h>
+#include <winrt/Windows.UI.Xaml.Documents.h>

```

Comment:
I think this should be in pch.h

Reply:
Notably, the Microsoft.UI.Xaml one _shouldn't_ be, because it's from a nuget package, but this one's in the SDK so it _should_ be.